README2.md

```diff
@@ -0,0 +1,23 @@
+# Chronologue Project
+----------------
+by [The Good Docs Project](thegooddocsproject.dev)
+
+Chronologue is an open source calendrical toolkit for time travel, including various utilities for developers working on time travel technology, including but not limited to time-travel devices (“time machines”), farviewers, and other IoT (internet of time) or “time-tech” applications, to convert between calendrical systems.
+
+Although the major calendrical systems (e.g. the Gregorian calendar, the Hijra calendar, Unix time) are adequate for most temporally-sequential use cases, they are not well-adapted for activities that involve traversing timestreams. Chronologue tools and libraries help developers convert between, among, and across traditional calendrical systems.
+
+Chronologue includes:
+
+* The Chronologue Time Standard (CTS), which provides consistent time coordinates across timelines
+* Test suites for integrations or applications using the Chronologue Time Standard
+* Tools for conversion between the Gregorian, Mayan, and Hijri calendars and the Chronologue Time Standard (CTS) 
+
```

Comment:
It'd be nice to know, probably right here, whether Chronologue supports self-crossing timestreams (for example—warning _The Good Place_ spoilers—the [Jeremy Bearimy](https://www.youtube.com/watch?v=RFm9ClqlGuo&feature=youtu.be&t=32) timeline). For simplicity, I'd prefer to declare such timestreams out of scope.